htdocs/societe/class/api_thirdparties.class.php

```diff
@@ -1170,6 +1171,129 @@ function deleteCompanyBankAccount($id, $socid){
 		return $account->delete(DolibarrApiAccess::$user);
 	}
 
+	/**
+	 * Generate a sepamandate Document
+	 * 
+	 * @param int $socid thirdparty id
+	 * @param int $id companybankid
+	 * 
+	 * @return array Check success
+	 * 
+	 * @url GET /generateMandat/{socid}
+	 */
+	public function generateMandat($socid, $id = null){
+
```

Comment:
@yolocodefrench can you write like this
`
public function generateMandat($socid, $id = null)
{
`